subsys/pm/device_system_managed.c

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2024 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/pm/device.h>
+#include <zephyr/pm/device_runtime.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(pm_device, CONFIG_PM_DEVICE_LOG_LEVEL);
+
+#define DT_PM_DEVICE_ENABLED(node_id)						\
+	COND_CODE_1(DT_PROP_OR(node_id, zephyr_pm_device_disabled, 0),	\
```

Comment:
Non blocking:
```suggestion
	COND_CODE_1(DT_PROP(node_id, zephyr_pm_device_disabled),	\
```
should be used when prop is a boolean